lib/Transforms/Tapir/OpenMPABI.cpp

```diff
@@ -488,19 +488,19 @@ Function* formatFunctionToTask(Function* extracted, CallInst* cal) {
   IRBuilder<> CallerIRBuilder(cal);
   auto *SharedsTySize =
       CallerIRBuilder.getInt64(DL.getTypeAllocSize(SharedsTy));
-  auto *KmpTaskTTy = createKmpTaskTTy(C);
+  //unused -- auto *KmpTaskTTy = createKmpTaskTTy(C);
```

Comment:
Can we just delete this line and the other `//unused` dead lines in this file?  Are we worried we'll need them later?